tests/unittests/Foundation/NSURLTests.m

```diff
@@ -190,6 +189,8 @@ void testNSURLMethod(SEL selector, NSURL* input, id argument, NSURL* expected) {
     ASSERT_OBJCEQ_MSG(fileURLString, newFileURLString, "File URLs do not match!");
 }
 
+#ifdef _WIN32
+#include "windows.h"
 TEST(NSURL, CheckResourceIsReachable) {
```

Comment:
oops

---
In reply to: [70349984](https://github.com/Microsoft/WinObjC/pull/634#discussion_r70349984) [](ancestors = 70349984)